src/angular/form-elements/dropdown/dropdown-models.ts

```diff
@@ -0,0 +1,17 @@
+/**
+ * Created by M.S.BIT on 19/11/2017.
+ */
+
+
+export enum DropDownOptionType {
+    Simple, //default
+    Header,
+    Disable,
+    HorizontalLine
+}
+
+export interface IDropDownOption {
+    value?:any;
+    label:string;
+    type?:number;
+}
```

Comment:
`value` - since this is what you emit when selecting an option, I think `value` should be mandatory attribute.
`label` - can be optional, and use `toString(value)` has default label if not exist.
`type` - mandatory, just set it to default `DropDownOptionType.Simple`.